rust_src/src/symbols.rs

```diff
@@ -404,5 +402,40 @@ pub fn symbol_value(symbol: LispSymbolRef) -> LispObject {
     }
     val
 }
+/// Non-nil if VARIABLE has a local binding in buffer BUFFER.
+/// BUFFER defaults to the current buffer.
+#[lisp_fn(min = "1")]
+pub fn local_variable_p(mut symbol: LispSymbolRef, buffer: LispBufferOrCurrent) -> bool {
+    let buf: LispBufferRef = buffer.into();
+
+    symbol = symbol.get_indirect_variable();
+
+    match symbol.get_redirect() {
```

Comment:
It seems we need here the case like `symbol_redirect::SYMBOL_VARALIAS => ...`.

Reply:
the line above `symbol = symbol.get_indirect_variable();` handles it

Reply:
I misunderstood, thanks